`DateTimeWithTimeZone` does not implement `Serialize` and `Deserialize`

Hi. `sea-orm` use `DateTimeWithTimeZone` in `Model` to map `TIMESTAMPTZ` in postgresql. `DateTimeWithTimeZone` is just an alias of type `chrono::DateTime<chrono::FixedOffset>`, which has already implemented `Serialze` and `Deserialize` traits in `chrono` crate. However, the compiler says `DateTimeWithTimeZone` does not implement `Serialize` and `Deserialize` in the generated scripts. It is weird. My rust version is `1.56.1`, and the `sea-orm` is synced with the `master` branch in this repository.

> Hey @YoshieraHuang, would you mind showing us the detail compile errors?

This is my scripts:
```rust
#[derive(Clone, Debug, PartialEq, DeriveEntityModel, Serialize, Deserialize)]
#[sea_orm(table_name = "materials")]
pub struct Model {
    #[sea_orm(primary_key)]
    pub id: i32,
    pub created_at: DateTimeWithTimeZone,
    pub updated_at: DateTimeWithTimeZone,
    pub name: String,
    #[sea_orm(column_type = "Text", nullable)]
    pub description: Option<String>,
    pub tag_ids: Vec<u8>,
}
```

And this is the error:
```
error[E0277]: the trait bound `chrono::datetime::DateTime<chrono::offset::fixed::FixedOffset>: Serialize` is not satisfied
    --> src/materials/entity/material_tags.rs:11:5
     |
11   |     pub created_at: DateTimeWithTimeZone,
     |     ^^^ the trait `Serialize` is not implemented for `chrono::datetime::DateTime<chrono::offset::fixed::FixedOffset>`
     |
note: required by `materials::_::_serde::ser::SerializeStruct::serialize_field`
    --> /home/jasper/.cargo/registry/src/github.com-1ecc6299db9ec823/serde-1.0.130/src/ser/mod.rs:1893:5
     |
1893 | /     fn serialize_field<T: ?Sized>(
1894 | |         &mut self,
1895 | |         key: &'static str,
1896 | |         value: &T,
1897 | |     ) -> Result<(), Self::Error>
1898 | |     where
1899 | |         T: Serialize;
     | |_____________________^

error[E0277]: the trait bound `chrono::datetime::DateTime<chrono::offset::fixed::FixedOffset>: Serialize` is not satisfied
    --> src/materials/entity/material_tags.rs:12:5
     |
12   |     pub updated_at: DateTimeWithTimeZone,
     |     ^^^ the trait `Serialize` is not implemented for `chrono::datetime::DateTime<chrono::offset::fixed::FixedOffset>`
     |
note: required by `materials::_::_serde::ser::SerializeStruct::serialize_field`
    --> /home/jasper/.cargo/registry/src/github.com-1ecc6299db9ec823/serde-1.0.130/src/ser/mod.rs:1893:5
     |
1893 | /     fn serialize_field<T: ?Sized>(
1894 | |         &mut self,
1895 | |         key: &'static str,
1896 | |         value: &T,
1897 | |     ) -> Result<(), Self::Error>
1898 | |     where
1899 | |         T: Serialize;
     | |_____________________^

error[E0277]: the trait bound `chrono::datetime::DateTime<chrono::offset::fixed::FixedOffset>: Deserialize<'_>` is not satisfied
    --> src/materials/entity/material_tags.rs:11:5
     |
11   |     pub created_at: DateTimeWithTimeZone,
     |     ^^^ the trait `Deserialize<'_>` is not implemented for `chrono::datetime::DateTime<chrono::offset::fixed::FixedOffset>`
     |
note: required by `next_element`
    --> /home/jasper/.cargo/registry/src/github.com-1ecc6299db9ec823/serde-1.0.130/src/de/mod.rs:1703:5
     |
1703 | /     fn next_element<T>(&mut self) -> Result<Option<T>, Self::Error>
1704 | |     where
1705 | |         T: Deserialize<'de>,
     | |____________________________^

error[E0277]: the trait bound `chrono::datetime::DateTime<chrono::offset::fixed::FixedOffset>: Deserialize<'_>` is not satisfied
    --> src/materials/entity/material_tags.rs:12:5
     |
12   |     pub updated_at: DateTimeWithTimeZone,
     |     ^^^ the trait `Deserialize<'_>` is not implemented for `chrono::datetime::DateTime<chrono::offset::fixed::FixedOffset>`
     |
note: required by `next_element`
    --> /home/jasper/.cargo/registry/src/github.com-1ecc6299db9ec823/serde-1.0.130/src/de/mod.rs:1703:5
     |
1703 | /     fn next_element<T>(&mut self) -> Result<Option<T>, Self::Error>
1704 | |     where
1705 | |         T: Deserialize<'de>,
     | |____________________________^

error[E0277]: the trait bound `chrono::datetime::DateTime<chrono::offset::fixed::FixedOffset>: Deserialize<'_>` is not satisfied
    --> src/materials/entity/material_tags.rs:11:5
     |
11   |     pub created_at: DateTimeWithTimeZone,
     |     ^^^ the trait `Deserialize<'_>` is not implemented for `chrono::datetime::DateTime<chrono::offset::fixed::FixedOffset>`
     |
note: required by `next_value`
    --> /home/jasper/.cargo/registry/src/github.com-1ecc6299db9ec823/serde-1.0.130/src/de/mod.rs:1842:5
     |
1842 | /     fn next_value<V>(&mut self) -> Result<V, Self::Error>
1843 | |     where
1844 | |         V: Deserialize<'de>,
     | |____________________________^

error[E0277]: the trait bound `chrono::datetime::DateTime<chrono::offset::fixed::FixedOffset>: Deserialize<'_>` is not satisfied
    --> src/materials/entity/material_tags.rs:12:5
     |
12   |     pub updated_at: DateTimeWithTimeZone,
     |     ^^^ the trait `Deserialize<'_>` is not implemented for `chrono::datetime::DateTime<chrono::offset::fixed::FixedOffset>`
     |
note: required by `next_value`
    --> /home/jasper/.cargo/registry/src/github.com-1ecc6299db9ec823/serde-1.0.130/src/de/mod.rs:1842:5
     |
1842 | /     fn next_value<V>(&mut self) -> Result<V, Self::Error>
1843 | |     where
1844 | |         V: Deserialize<'de>,
     | |____________________________^
```

--- a/.github/workflows/rust.yml
+++ b/.github/workflows/rust.yml
@@ -314,7 +314,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest]
-        path: [86, 249, 262]
+        path: [86, 249, 262, 319]
     steps:
       - uses: actions/checkout@v2
 

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -63,7 +63,7 @@ default = [
 ]
 macros = ["sea-orm-macros"]
 mock = []
-with-json = ["serde_json", "sea-query/with-json"]
+with-json = ["serde_json", "sea-query/with-json", "chrono/serde"]
 with-chrono = ["chrono", "sea-query/with-chrono"]
 with-rust_decimal = ["rust_decimal", "sea-query/with-rust_decimal"]
 with-uuid = ["uuid", "sea-query/with-uuid"]

--- a/issues/319/Cargo.toml
+++ b/issues/319/Cargo.toml
@@ -1,0 +1,18 @@
+[workspace]
+# A separate workspace
+
+[package]
+name = "sea-orm-issues-319"
+version = "0.1.0"
+edition = "2021"
+publish = false
+
+[dependencies]
+async-std = { version = "^1", features = ["attributes"] }
+serde = { version = "^1", features = ["derive"] }
+sea-orm = { path = "../../", features = [
+    "sqlx-mysql",
+    "runtime-async-std-native-tls",
+    "with-json",
+    "macros",
+], default-features = false }

--- a/issues/319/src/main.rs
+++ b/issues/319/src/main.rs
@@ -1,0 +1,14 @@
+mod material;
+use sea_orm::*;
+
+#[async_std::main]
+pub async fn main() {
+    let db = Database::connect("mysql://sea:sea@localhost/bakery")
+        .await
+        .unwrap();
+
+    async_std::task::spawn(async move {
+        material::Entity::find().one(&db).await.unwrap();
+    })
+    .await;
+}

--- a/issues/319/src/material.rs
+++ b/issues/319/src/material.rs
@@ -1,0 +1,20 @@
+use sea_orm::entity::prelude::*;
+use serde::{Serialize, Deserialize};
+
+#[derive(Clone, Debug, PartialEq, DeriveEntityModel, Serialize, Deserialize)]
+#[sea_orm(table_name = "materials")]
+pub struct Model {
+    #[sea_orm(primary_key)]
+    pub id: i32,
+    pub created_at: DateTimeWithTimeZone,
+    pub updated_at: DateTimeWithTimeZone,
+    pub name: String,
+    #[sea_orm(column_type = "Text", nullable)]
+    pub description: Option<String>,
+    pub tag_ids: Vec<u8>,
+}
+
+#[derive(Copy, Clone, Debug, EnumIter, DeriveRelation)]
+pub enum Relation {}
+
+impl ActiveModelBehavior for ActiveModel {}

